python/source/org/libreoffice/ide/eclipse/python/PythonProjectHandler.java

```diff
@@ -0,0 +1,356 @@
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.launching.JavaRuntime;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.wizards.newresource.BasicNewProjectResourceWizard;
+import org.eclipse.ui.wizards.newresource.BasicNewResourceWizard;
+import org.libreoffice.ide.eclipse.core.PluginLogger;
+import org.libreoffice.ide.eclipse.core.internal.helpers.UnoidlProjectHelper;
+import org.libreoffice.ide.eclipse.core.model.IUnoFactoryConstants;
+import org.libreoffice.ide.eclipse.core.model.IUnoidlProject;
+import org.libreoffice.ide.eclipse.core.model.ProjectsManager;
+import org.libreoffice.ide.eclipse.core.model.UnoFactoryData;
+import org.libreoffice.ide.eclipse.core.model.config.IOOo;
+import org.libreoffice.ide.eclipse.core.model.language.IProjectHandler;
+import org.libreoffice.ide.eclipse.core.utils.WorkbenchHelper;
+import org.libreoffice.ide.eclipse.python.utils.TemplatesHelper;
+import org.w3c.dom.Node;
+import org.xml.sax.InputSource;
+
+/**
+ * The Project handler implementation for Python.
+ */
+public class PythonProjectHandler implements IProjectHandler {
+
+    private static final String P_REGISTRATION_CLASSNAME = "regclassname"; //$NON-NLS-1$
+    private static final String P_JAVA_VERSION = "javaversion"; //$NON-NLS-1$
+
+    private static final String SOURCE_BASIS = "/source"; //$NON-NLS-1$
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void addOOoDependencies(IOOo pOoo, IProject pProject) {
+
+        //        IJavaProject javaProject = JavaCore.create(pProject);
+        //
+        //        OOoContainerPage.addOOoDependencies(pOoo, javaProject);
+        PluginLogger.debug("For a Python project 'No' OOo dependencies are added"); //$NON-NLS-1$
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void addProjectNature(IProject pProject) {
+        //        try {
+        //            if (!pProject.exists()) {
+        //                pProject.create(null);
+        //                PluginLogger.debug("Project created during language specific operation"); //$NON-NLS-1$
+        //            }
+        //
+        //            if (!pProject.isOpen()) {
+        //                pProject.open(null);
+        //                PluginLogger.debug("Project opened"); //$NON-NLS-1$
+        //            }
+        //
+        //            IProjectDescription description = pProject.getDescription();
+        //            String[] natureIds = description.getNatureIds();
+        //            String[] newNatureIds = new String[natureIds.length + 1];
+        //            System.arraycopy(natureIds, 0, newNatureIds, 0, natureIds.length);
+        //
+        //            // Adding the nature
+        //            newNatureIds[natureIds.length] = JavaCore.NATURE_ID;
```

Comment:
Can you add the python nature instead? The ID is "org.python.pydev.pythonNature", see https://stackoverflow.com/questions/21134945/how-to-add-the-python-nature-to-a-project-imported-from-github

We might also depend on PyDev when installing the LOEclipse python plugin.

Reply:
The required edits to the .project file has been made. Referenced from : http://www.pydev.org/faq.html#ImportExistingSources